src/platform/forms-system/src/js/widgets/DateWidget.jsx

```diff
@@ -87,6 +87,7 @@ export default class DateWidget extends React.Component {
             Month
           </label>
           <select
+            aria-describedby={this.props.id}
```

Comment:
The id isn't a value that would be useful for a user to hear, I don't think. I think we'd want the label, but labels might not be passed to widgets. So we might need to figure out how to pass labels to widgets or convert this to a field instead of a widget.

Reply:
Ideally we want the `<legend>` to have a relevant ID and point to that. I'd like the select to read out its own label "Month" in this case, pause, then read out the legend text. Having that additional context for what date we're asking users to put in (service start date, service end date, birth date) is critical for screen readers, especially when we have multiple date fieldsets on the page.

Reply:
From my understanding the id is connected to the legend id and that is what it should read out. I think `aria-describedby` is suppose to read out the element which the id is attached to. I never got this to work on Chrome so I wasn't sure if it worked the way @1Copenut wanted it to. @1Copenut were you able to try it out on JAWS?

Reply:
Ah yeah, looks like I misunderstood. This would actually need to be the id with `-label` appended to it.